Plugins~/Src/MeshSyncClientBlender/msblenContext.cpp

```diff
@@ -802,7 +802,7 @@ void msblenContext::doExtractMeshData(msblenContextState& state, BlenderSyncSett
             doExtractNonEditMeshData(state, settings, dst, obj, data);
         }
 
-        if (!settings.BakeModifiers&& !is_editing) {
+        if (!settings.BakeModifiers && !is_editing && state.manager.needsMirrorBaking()) {
```

Comment:
This code seems to be run when NOT baking, so the name `needsMirrorBaking` seems to be misleading ?

Reply:
I renamed it now.